l1-contracts/src/core/libraries/HeaderLib.sol

```diff
@@ -126,8 +130,8 @@ library HeaderLib {
    * @return The decoded header
    */
   function decode(bytes calldata _header) internal pure returns (Header memory) {
-    if (_header.length != 376) {
-      revert Errors.HeaderLib__InvalidHeaderSize(376, _header.length);
+    if (_header.length != 0x1ac) {
```

Comment:
Magic numbers 🧙 Can you add make a constant for it instead?

Reply:
Good idea. Addressed in https://github.com/AztecProtocol/aztec-packages/pull/4421/commits/9fc93087435005ac31fb2d3a177527d34ac4871d